sdk/monitor/opentelemetry-exporter-azure-monitor/src/config.ts

```diff
@@ -22,11 +24,14 @@ export interface AzureExporterInternalConfig {
   maxConsecutiveFailuresBeforeWarning: number;
 
   endpointUrl: string;
+
+  serviceApi: string;
 }
 
 export const DEFAULT_EXPORTER_CONFIG: AzureExporterInternalConfig = {
   instrumentationKey: "",
   endpointUrl: DEFAULT_BREEZE_ENDPOINT,
   batchSendRetryIntervalMs: 60_000,
-  maxConsecutiveFailuresBeforeWarning: 10
+  maxConsecutiveFailuresBeforeWarning: 10,
+  serviceApi: "2"
```

Comment:
Magic numbers?

Is there an enum or constant for these?

Reply:
Or is this supposed to be ```serviceApiVersion```?

Reply:
Added constants